sources/packages/backend/apps/api/test/jest-e2e.json

```diff
@@ -10,7 +10,7 @@
   "transform": {
     "^.+\\.(t|j)s$": "ts-jest"
   },
-  "testTimeout": 60000,
+  "testTimeout": 120000,
```

Comment:
Should we assume that it was pretty much executed at the limit of the 60s since the newly raised PRs after this were not facing the same issue so far?
We can maybe leave it at 90000.

Reply:
Yes, I agree that is weird that happened only now with this PR. Changed to 90s.